### ISSUE TITLE
[Bug]: Broken-apart animated model text breaks all words for screen reader

### ISSUE THREAD

**Issue by gmerritt** [danny-avila/LibreChat#7145]
### What happened?

The feature that animates model name in the center of a new chat window `b r e a k s    a p a r t    t e x t    i n t o    s e p a r a t e     e l e m e n t s 
  b y    l e t t e r`; screen readers cannot recognize the source words, and read each letter individually.

(Note also that more-complex characters, such as many emojis, get broken up by the animator.)

### Version Information

Active in [this commit](https://github.com/danny-avila/LibreChat/commit/7f1d01c35a7c9a916dc6d33f9f3915b4a452b4ad) -- April 23, 2025 -- and presumably present since animation was introduced several weeks ago.

### Steps to Reproduce

1. Start a new chat.

### What browsers are you seeing the problem on?

Chrome, Firefox

### Relevant log output

```shell
Screencast upload not working; will try to upload in a comment after posting.
```

### Screenshots

UC Berkeley [Web Access Analyst Lucy Greco](https://www.berkeley.edu/directory/?search-term=Greco,%20Lucy&uid=264621) may be contacted if additional information is needed.

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by gmerritt** [comment]
[Short screencast video](https://drive.google.com/file/d/1TlB2pvKq3uTlRRvfRGYRn9bWMtpJC02j/view?usp=sharing) featuring screen reader trying to read the `a n i m a t e d    c h a r a c t e r s`
